testsuite/tests/lib-bigarray/change_layout.ml

```diff
@@ -4,7 +4,7 @@ open Bigarray
 
 let pp_sep ppf () = Format.fprintf ppf ";@ "
  let print_array pp ppf a =
- Format.fprintf ppf "@[<hov>⟦%a⟧@]"
+ Format.fprintf ppf "@[<hov>\xe2\x9f\xa6%a\xe2\x9f\xa7@]"
```

Comment:
May I suggest to replace the format string with a simple "[|%a|]"? (This is the failing path of the test, so there would be no need to update the reference file).